decidim-conferences/app/views/decidim/conferences/admin/conference_copies/new.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_decidim_page_title(t("duplicate_conference", scope: "decidim.admin.titles", title: translated_attribute(current_conference.title))) %>
```

Comment:
For consistency with the rest of the actions, I'd drop the title of the Spaces in the HTML title. If we add it here we should also add it in all the other places  (like `edit.html.erb`)  of all the other spaces (Processes, Assemblies, etc) 

```suggestion
<% add_decidim_page_title(t("duplicate_conference", scope: "decidim.admin.titles")) %>
```